EEBOS/SCROWL
# Chore/notarize app

### Short summary

- the path to notarize script was wrong
- the notarize script needs to be in commonjs format

---

### Test steps

1. `yarn run deploy` the app should build/package without errors. Output is in `apps/course-authoring/release/build`
